package.json

```diff
@@ -6,7 +6,8 @@
     "build": "skyux build-public-library",
     "start": "skyux serve --hmr",
     "test": "skyux test --coverage library --logFormat none",
-    "test:visual": "skyux e2e --logFormat none"
+    "test:visual": "skyux e2e --logFormat none",
+    "postinstall": "ngcc"
```

Comment:
☝️ This was necessary to get `skyux build` to work.
Source: https://stackoverflow.com/a/63727427/91631
Can we get away with this?

Reply:
I'm not 100% sure about this exact issue. However, I believe that `@circlon/angular-tree-component` version 11 requires at least Angular 12 and this still has Angular 9 being pulled in. Could that be contributing?

Reply:
Could we use the version of `@circlon/angular-tree-component` that works with Angular 9? At least until we can install Angular 12 locally. Otherwise, we'll have to wait to make this change until we can run a migration of this library.

Reply:
Ok, after downgrading to `@circlon/angular-tree-component@^9.0.0` (the Angular 9 version), everything worked fine and I no longer needed to add this post install step. Removed it!